projects/plugins/jetpack/extensions/blocks/subscriptions/subscriptions.php

```diff
@@ -599,19 +596,22 @@ function get_post_access_level_for_current_post() {
 }
 
 /**
- * Renders the WP.com version of the subscriptions block.
+ * Renders the subscriptions block at the site.
  *
  * @param array $data    Array containing block view data.
  * @param array $classes Array containing the classes for different block elements.
  * @param array $styles  Array containing the styles for different block elements.
  *
  * @return string
  */
-function render_wpcom_subscribe_form( $data, $classes, $styles ) {
-	global $current_blog;
-
-	$form_id = 'subscribe-blog' . ( Jetpack_Subscriptions_Widget::$instance_count > 1 ? '-' . Jetpack_Subscriptions_Widget::$instance_count : '' );
-	$url     = defined( 'SUBSCRIBE_BLOG_URL' ) ? SUBSCRIBE_BLOG_URL : '';
+function render_for_website( $data, $classes, $styles ) {
+	$blog_id            = \Jetpack_Options::get_option( 'id' );
+	$form_id            = 'subscribe-blog-' . $data['widget_id'];
```

Comment:
On WPCOM, form id was always "subscribe-blog". But I don't think it matters here.

Reply:
On Jetpack we had:

```
subscribe-blog
subscribe-blog-2
subscribe-blog-3
```

etc. The same for `$subscribe_field_id`.

I basically simplified and unified it a bit. It just tries to avoid multiple same IDs on the same page.

Reply:
tried to clean up element ids with https://github.com/Automattic/jetpack/pull/33865/commits/dfbf7e3c537fc0ce786cffcce2044c8a22096e6c